### ISSUE TITLE
RTL Direction in markdown viewer

### ISSUE THREAD

**Issue by yazdan** [tagspaces/tagspaces#586]
Hi,

First of all great product. 

My note taking process involves sometimes writing in Persian and Persian is RTL language.
Is there a plan to add support for such languages.

This also can be resolved if there is any special css or Markdown tag or even TOML to specify "dir:rtl" for this special markdown document?

Thanks again

**Comment by uggrock** [comment]
Hi,

all the delivered extensions used for viewing and editing files are open source and available on github. So if you are familiar with programming feel free to create pull request for this feature. Repositories for some of these extensions are listed bellow:

https://github.com/tagspaces/viewerMD/
https://github.com/tagspaces/viewerHTML
https://github.com/tagspaces/editorText

regards,
Ilian

**Comment by yazdan** [comment]
Hi,

Thanks for your response. 

I'm not that familiar with Javascript and related issues but I try to resolve it. As far as I've investigated the issue, editorText uses codemirror. Codemirror support bidi languages (Persian/Arabic/Hebrew) and it has two related options which are 

- rtlMoveVisually
- direction

I've set these two properties with different values in editorText's `main.js` and reloaded the application using F5 and nothing happened. 

I'll investigate this more and let you know about the results.
